safir/src/safir/uws/_handlers.py

```diff
@@ -127,11 +128,12 @@ async def get_job(
     uws_factory: Annotated[UWSFactory, Depends(uws_dependency)],
 ) -> Response:
```

Comment:
Oh, huh, I was assuming that this would give you a schema that was only suitable for JSON and not for XML, but it looks like the XML isn't that horrible. I'll go ahead and do this; at the least, it doesn't seem like it would hurt. Thanks!